app/ui/pages/10_Settings.py

```diff
@@ -276,3 +283,16 @@ def expander(num: int, title: str) -> st.expander:
         connection.execute("call internal.refresh_queries(true);")
         bar.progress(100, text="All events refreshed.")
         msg.info("Reset Complete.")
+
+with initial_probes:
+    st.title("Pre-Configured Probes and Labels")
+    do_reset = st.button("Re-load all preconfigured labels and probes.")
```

Comment:
```suggestion
    st.title("Pre-Configured Probes and Labels")
    st.text("OpsCenter includes a number of helpful labels and probes that are created when you have none defined. OpsCenter will not overwrite any existing labels and probes you have defined. Click the button below to reset all labels and probes to the defaults provided by OpsCenter.")
    do_reset = st.button("Re-load all preconfigured labels and probes.")
```

Something like this to warn that this is destructive and explain why you might want to click this button?